### PR TITLE
Add the 0918 meetup agenda

### PR DESCRIPTION
**Pull request by ThreeMonth03**
This pr is to add the meetup agenda to the sciwork website.

### CODE REVIEW

File: content/pages/meetup/2024/0918-nycu.rst [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+| 19:30 ~ 21:00   | free chat                | modmesh discussion and training :                     | nsd teaching assistance :                    |
+|                 |                          |                                                       |                                              |
+|                 |                          | 1. Use Bezier curve to fit NACA 4-digit airfoil       | 1. project discussion                        |
+|                 |                          | 2. Prototype profiler object serialization using JSON | 2. homework assistance                       |
+|                 |                          | 3. Build modmesh                                      |                                              |
+|                 |                          | 4. Make a simple PR for modmesh                       |                                              |
++-----------------+--------------------------+-------------------------------------------------------+----------------------------------------------+
```
Comment (ThreeMonth03) [inline content/pages/meetup/2024/0918-nycu.rst:33]: 1. I list the detail for every topic.
2. I add the topic about nsd assistance.

File: content/pages/meetup/2024/0918-nycu.rst [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+
+numerical software development (nsd) teaching assistance
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+This session is for the students who are taking the 
+`numerical software development course <https://yyc.solvcon.net/en/latest/nsd/index.html>`__.
+If you have any questions about the homework or the project,
+you can come to the meetup and discuss the questions with the teaching assistant.
+
```
Comment (ThreeMonth03) [inline content/pages/meetup/2024/0918-nycu.rst:88]: The description of nsd TA.

**Review by ThreeMonth03 — COMMENTED** [review]
@yungyuc @YenPeiChen07 Please review the pr when you are available. Thank you very much.

**Comment by YenPeiChen07** [comment]
> @yungyuc @YenPeiChen07 Please review the pr when you are available. Thank you very much.

It looks great to me.

**Comment by ThreeMonth03** [comment]
> > @yungyuc @YenPeiChen07 Please review the pr when you are available. Thank you very much.
> 
> It looks great to me.

Thanks, I decide to merge this pr. If there is any suggestions afterwards, we could make the follow-up pull request.

**Pull request merged**